modules/battery_manager/jsx/editEntryIndex.js

```diff
@@ -0,0 +1,30 @@
+/** global ReactDOM
+ *
+ * Battery Manager Edit Form
+ *
+ * Module component rendering Edit form
+ *
+ * @author Victoria Foing
+ *
+ */
+import BatteryManagerEditForm from './editEntry';
```

Comment:
1. it seems to me that there should really be only 1 index file that renders all the other components. 
2. if this index file only imports 1 component (editEntry), then I believe the call to render can just be appended to the end of the editEntry class